dotCMS/src/main/java/com/dotcms/rendering/velocity/viewtools/content/util/ContentUtils.java

```diff
@@ -181,9 +211,9 @@ public static Contentlet find(String inodeOrIdentifier, User user, String tmDate
                             .replaceAll("\\+working\\:true", "");
                     final String formatedDate = ESMappingAPIImpl.datetimeFormat.format(futureDate);
 
-                    final String notExpired = " +expdate_dotraw:[" + formatedDate + " TO 29990101000000] ";
+                    final String notExpired = " +expdate:[" + formatedDate + " TO 29990101000000] ";
```

Comment:
is there any test for this change? Please be sure it doesn't break other logic

Reply:
done

Reply:
The original issue reported was still failing so that is why this change